test/downloadGTest.cmake.in

```diff
@@ -0,0 +1,25 @@
+
+############################################################################
+# Copyright (c) Johan Mabille, Sylvain Corlay and Wolf Vollprecht          #
+# Copyright (c) QuantStack                                                 #
+#                                                                          #
+# Distributed under the terms of the BSD 3-Clause License.                 #
+#                                                                          #
+# The full license is in the file LICENSE, distributed with this software. #
+############################################################################
+
+cmake_minimum_required(VERSION 2.8.2)
+
+project(googletest-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(googletest
+    GIT_REPOSITORY    https://github.com/JohanMabille/googletest.git
```

Comment:
let's not include this. I really think we should just fix gtest package or add gmock.

Reply:
I also prefer using the packaged version of gtest. We should fix upstream, I don't like having a dep to something we don't need

Reply:
It's now fixed without installing from sources. Just modified to use `add_library` instead of `find_package`.